migrations/20200514164200-add-collective-index-for-incognito-query.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Enhance User.prototype.getIncognitoProfile() performance
+    await queryInterface.sequelize.query(`
+      CREATE INDEX IF NOT EXISTS collectives_incognito_query ON "public"."Collectives"("CreatedByUserId");
```

Comment:
I would use a more generic name for the index.

Reply:
Cool.
Renamed using the standard field name for the index, I'm keeping the comment as paper-trail.